spec/rubocop/cli/cli_autocorrect_spec.rb

```diff
@@ -1693,4 +1693,41 @@ def self.some_method(foo, bar: 1)
     expect(status).to eq(0)
     expect(source_file.read).to eq(source)
   end
+
+  it 'corrects multiple Layout/DotPosition offenses' do
```

Comment:
You can use `expect_correction` in your new spec on the cop's spec rather than having it be tested through the CLI spec.